### DISCUSSION TITLE
Adding powerlines

### DISCUSSION THREAD

**Discussion by ForgotMyPasswd** [JakeStanger/ironbar#540] [General]
I'm trying to create powerlines, similar to this
![image](https://github.com/JakeStanger/ironbar/assets/96489361/e05f5a3e-cc08-43a6-bfb0-2a1eada21310)
I'm not exactly sure how to achieve this though, I'm using custom labels with nerd font icons to create dividers which doesn't work since I can't remove the padding from the top and bottom. Any advice?

**Comment by JakeStanger** [comment]
Hey, apologies for the delay in getting back to you, but I've just had a chance to play with this. 

You can indeed do this with a label, and a particular Nerd Font icon that isn't too obvious called `nf-pl-left_hard_divider` (it's a bit taller than the arrows).

Using that you can do something like:

```corn
let {
  $clock = { type = "clock" }
  $pl_arrow = { type = "label" label ="" class="pl-arrow" }
} in {
  start = [ $clock $pl_arrow ]
}
```

then style with:

```css
.clock {
  background-color: red;
}

.pl-arrow {
   color: red;
   font-size: 32px;
}
```

(you will need to adjust font size based on your bar font & height)
